pkg/core/system.go

```diff
@@ -191,6 +191,9 @@ func (kc *kubectlClient) SystemUninstall(options SystemUninstallOptions) (bool,
 		if err != nil {
 			return false, err
 		}
+		deleteNamespaceResource(kc, "istio-system", "service", "knative-ingressgateway")
```

Comment:
Why are these resources not deleted when the `istio-system` namespace is deleted? Or is there an ordering requirement?

I'd recommend logging an issue to record the requirements for this PR.

Reply:
forgot to add "Fixes", fixed now, the issue is #1002

Reply:
Ah, lovely. Thanks. Makes sense now.